assets/css/style.css

```
@@ -0,0 +1,479 @@
+/* Simple CSS Reset */
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+/* Main Container */
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  gap: 10px;
+}
+
+/* Header (Navbar) */
+
+.header {
+  min-height: 43px;
+  padding: 0 20px;
+  background-color: #fff;
+  border-bottom: 1px solid #ddd;
+  grid-column: 1 / 13;
+  position: sticky;
+  top: 0;
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  align-items: center;
+}
+
+.header a {
+  text-decoration: none;
+  color: #222;
+}
+
+.nav-left {
+  grid-column: 1 / 2;
+  justify-self: left;
+}
+
+.nav-left .material-icons {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.menu-icon,
+.search-icon {
+  padding: 5px;
+  margin-right: 20px;
+}
+
+.category {
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 15px;
+}
+
+.category:hover {
+  text-decoration: underline;
+}
+
+.nav-logo {
+  grid-column: 2 / 3;
+  justify-self: center;
+}
+
+.nav-logo img {
+  width: 220px;
+}
+
+.nav-right {
+  grid-column: 3 / 4;
+  justify-self: right;
+}
+
+.btn {
+  padding: 7px 20px;
+  background-color: #567b95;
+  cursor: pointer;
+  color: #fff;
+  border-radius: 5px;
+  font-weight: 700;
+  border: 1px solid #ddd;
+}
+
+.btn:hover {
+  background-color: #326891;
+}
+
+.btn-block {
+  width: 55%;
+}
+
+/* Main content */
+
+.main {
+  min-height: 1024px;
+  grid-column: 1 / 13;
+  max-width: 950px;
+  margin: 0 auto 50px;
+}
+
+.main-title {
+  margin: 25px auto 35px !important;
+}
+
+.main-title h1 {
+  font-size: 40px;
+  text-align: left;
+}
+
+.banner img {
+  max-width: 100%;
```

Why would you use `max-width`?
```suggestion
  display: block;
```